sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventController.java

```diff
@@ -591,7 +591,7 @@ public CommitDiscardWrapperComponent<EventDataForm> getEventCreateComponent(Case
 			caseDataDto = FacadeProvider.getCaseFacade().getCaseDataByUuid(caseRef.getUuid());
 		}
 
-		EventDataForm eventCreateForm = new EventDataForm(true, false);
+		EventDataForm eventCreateForm = new EventDataForm(true, false, true);
```

Comment:
Again it is very hard for an outsider to understand the reasoning why we set these values always to a certain value. I would add a short comment.